components/datetime/tests/fixtures/tests/components.json

```diff
@@ -27,6 +27,24 @@
             }
         }
     },
+    {
+        "input": {
+            "value": "2022-05-03T14:15:07.123",
```

Comment:
Question: Is it expected that a fractional second of 0 should default to 3 instead? This seems a little odd behavior.